sqlite_fdw.c

```diff
@@ -3143,14 +3145,14 @@ sqliteImportForeignSchema(ImportForeignSchemaStmt *stmt,
 				bool		not_null;
 				char	   *default_val;
 				int			primary_key;
-				
-				rc = sqlite3_step(pragma_stmt);
-				if (rc == SQLITE_DONE)
+				int			rc1 = sqlite3_step(pragma_stmt);
```

Comment:
Is this fix for gcc warning?    
Currently we are using gcc4 (default of CentOS 7) and have no warning for it.   
Could you please share your env spec?

Reply:
> Is this fix for gcc warning?

Yes. This and some nearest changes with variable names.

> Could you please share your env spec?

Yes. My output `gcc -v` shows more newer version, **`gcc 10.2.1 20210110`**. Also this warning was only during PostgreSQL 16 compile process.

```
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/10/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:amdgcn-amdhsa:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Thread model: posix
Supported LTO compression algorithms: zlib zstd
gcc version 10.2.1 20210110
```

Reply:
Thank you for share. Please help me resolve this thread.